### ISSUE TITLE
[idea] Remove the Trigger when Found Nothing

### ISSUE THREAD

**Issue by kookma** [EvidentlyCube/TW5-TiddlerCompletion#8]
Autocomplete looks for a match when the trigger starts, but what if nothing find?

See below screetcast
![img_180_msedge](https://user-images.githubusercontent.com/830394/210930708-eff4d63b-cea0-47e5-a3e6-c37b3d2ec94e.gif)

**Expected behaviour**
* If nothing found one presses escape ot enter
* if pressed enter, then keep what entered and REMOVE the trigger chars
* if pressed escape, delete everything

For example, if `res:` is a trigger as shown above, if one types `res:Telumire` and `Telumire` is not found, then on enter, keep the word (what has been types) and remove the `res:` the same behaviour if Autocomplete finds the entry.

**Comment by EvidentlyCube** [comment]
That would be a neat option but certainly not a default behavior. If you have `[[` for a trigger to add links you don't want the initial `[[` to be removed if you cancel/hit enter on empty.

**Comment by kookma** [comment]
> If you have `[[` for a trigger to add links you don't want the initial `[[` to be removed if you cancel/hit enter on empty.

That is quite true! But there are cases you just type those words for triggering like `emoji:`

> That would be a neat option

That is fair to have this optional.
